hpke.go

```diff
@@ -184,21 +184,38 @@ func keySchedule(suite CipherSuite, mode Mode, sharedSecret, info, psk, pskID []
 	return params, nil
 }
 
+// Context represents an HPKE context encoded on the wire.
+type Context struct {
```

Comment:
Seems like this doesn't need to be public / upper-case.

Reply:
Done.